### PR TITLE
Route, readme and policy changes

### PR DESCRIPTION
**Pull request by AndreasElia**

### CODE REVIEW

File: src/Console/Stubs/DiscussionPolicy.stub [hunk 1]
```diff
@@ -1,13 +1,9 @@
 <?php
 
-namespace AndreasElia\Forum\Policies;
-
-use Illuminate\Auth\Access\HandlesAuthorization;
+namespace DummyNamespace;
 
 class PostPolicy
 {
-    use HandlesAuthorization;
```
Comment (Dylan-DPC) [inline src/Console/Stubs/DiscussionPolicy.stub]: this is needed in stub

File: src/Console/Stubs/DiscussionPolicy.stub [hunk 2]
```diff
@@ -1,13 +1,9 @@
 <?php
 
-namespace AndreasElia\Forum\Policies;
-
-use Illuminate\Auth\Access\HandlesAuthorization;
+namespace DummyNamespace;
 
 class PostPolicy
```
Comment (Dylan-DPC) [inline src/Console/Stubs/DiscussionPolicy.stub]: should be `DiscussionPolicy` and not `PostPolicy`

File: src/Console/Stubs/GroupPolicy.stub [hunk 3]
```diff
@@ -1,13 +1,9 @@
 <?php
 
-namespace AndreasElia\Forum\Policies;
-
-use Illuminate\Auth\Access\HandlesAuthorization;
+namespace DummyNamespace;
 
 class GroupPolicy
 {
-    use HandlesAuthorization;
```
Comment (Dylan-DPC) [inline src/Console/Stubs/GroupPolicy.stub]: needed in stub

File: src/Console/Stubs/PostPolicy.stub [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
 
-namespace AndreasElia\Forum\Policies;
+namespace DummyNamespace;
 
-use Illuminate\Auth\Access\HandlesAuthorization;
-
-class DiscussionPolicy
+class PostPolicy
 {
-    use HandlesAuthorization;
```
Comment (Dylan-DPC) [inline src/Console/Stubs/PostPolicy.stub]: needed in stub

File: src/Console/Stubs/DiscussionPolicy.stub [hunk 5]
```diff
@@ -2,7 +2,9 @@
 
 namespace DummyNamespace;
 
-class PostPolicy
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class DiscussionPolicy
```
Comment (Dylan-DPC) [inline src/Console/Stubs/DiscussionPolicy.stub]: handles authorization trait?

**Comment by ntzm** [comment]
I feel like `DELETE` requests should return `204 No Content`

**Pull request merged**